mlrun/runtimes/base.py

```diff
@@ -976,6 +992,46 @@ def _update_run_state(
 
         return resp
 
+    def _save_or_fire_notifications(self, runobj: RunObject, local: bool = False):
```

Comment:
```suggestion
    def _save_or_fire_notifications(self, runobj: RunObject):
```

Not needed anymore as you're using `is_running_as_api`.